repo.json

```diff
@@ -3607,6 +3607,21 @@
           "version": "1.0.8",
           "commit": "d244fe87186d4ec2bffbe3fe0fb1c331fc63b60e",
           "url": "https://github.com/doitintl/bigquery-grafana"
+        },
+        {
+          "version": "2.0.0",
```

Comment:
This is not a valid release, would you mind removing it from the PR?

Reply:
No problem